scripts/test-w-coverage.sh

```diff
@@ -16,6 +16,7 @@ go test -timeout 120s \
    -failfast \
    -race \
    -v \
+   -coverpkg=./pkg/...,./cmd/... \
```

Comment:
Why is this addition required?
Could you add a sentence in the commit description regarding this for additional context.